src/baxter_actions.cpp

```diff
@@ -0,0 +1,155 @@
+#include "rapid_pbd/baxter_actions.h"
+
+#include <string>
+#include <typeinfo>
+
+#include "actionlib/client/simple_action_client.h"
+#include "actionlib/server/simple_action_server.h"
+#include "control_msgs/FollowJointTrajectoryAction.h"
+#include "control_msgs/GripperCommandAction.h"
+#include "control_msgs/SingleJointPositionAction.h"
+#include "controller_manager_msgs/ListControllers.h"
```

Comment:
ListControllers, SwitchController, ArmControllerState, FreezeArm, and RelaxArm are all unused. Delete.